src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -111,126 +112,75 @@ const HealthCardDisplay = () => {
   const history = useHistory();
   const { t } = useTranslation();
   const {
-    qrCodes, healthCardVerified, issuerVerified, validPrimarySeries
+    qrError,
+    jws,
```

Comment:
```suggestion
    jws,
    resetQrCodes
```

Reply:
I'm also seeing the React warning for not including `resetQrCodes` in the useEffect dependency array.